example/lib/meeting/meeting_page.dart

```diff
@@ -231,24 +231,40 @@ class _MeetingPageState extends State<MeetingPage> with WidgetsBindingObserver {
         }
         setState(() {});
         break;
+      case 6:
```

Comment:
remove these comments , lets try to keep example app cleaner

Reply:
Sure,this code was for active speaker mode so for now I have commented it.